src/main/java/com/unzer/payment/paymenttypes/Invoice.java

```diff
@@ -35,6 +35,7 @@
  * @author Unzer E-Com GmbH
  *
  */
+@Deprecated
```

Comment:
I think only invoice secured should be marked as deprecated but not the normal invoice.